src/plugins/gc-subway/_base.scss

```
@@ -41,21 +90,25 @@
 					transition: width .2s, height .2s, left .2s, margin-top .2s;
 					width: 1.2em;
 				}
-				&.active > :first-child::before {
+
+				&.active> :first-child::before {
```

add a space before "<"
```suggestion
				&.active > :first-child::before {
```